src/main/java/seedu/duke/exceptions/OutOfRangeException.java

```diff
@@ -5,7 +5,7 @@
 public class OutOfRangeException extends Exception{
     public void printOutOfRange() {
         System.out.println(LINE);
-        System.out.println("The number you have entered is out of range! (Accepted range is 0 to 99999999)");
+        System.out.println("The number you have entered exceeds the maximum limit of 99999999.");
```

Comment:
Small remark actually, but would 99,999,999 be easier to see? similar to ug

Reply:
Changed to match UG format